exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Constraints.kt

```diff
@@ -111,9 +111,10 @@ data class ForeignKeyConstraint(
     /** Name of this constraint. */
     val fkName: String
         get() = tx.db.identifierManager.cutIfNecessaryAndQuote(
-            name ?: "fk_${fromTable.tableNameWithoutSchemeSanitized}_${
-                from.joinToString("_") { it.name }
-            }__${target.joinToString("_") { it.name }}"
+            name ?: (
+                "fk_${fromTable.tableNameWithoutSchemeSanitized}_${from.joinToString("_") { it.name }}__" +
+                    target.joinToString("_") { it.name }
+                )
```

Comment:
Everything looks good. This is the only one that isn't showing up for me locally as an error/warning, so I'm not sure if there's still something not syncing up with the detekt configuration.

Reply:
It shows up when you run `./gradlew detekt`
<img width="1134" alt="Screenshot 2023-07-17 at 11 01 08 AM" src="https://github.com/JetBrains/Exposed/assets/38375996/321ca89e-f398-4c5a-a11a-44466dfceb5a">